diff --git a/node_modules/@epubjs-react-native/core/lib/commonjs/View.js b/node_modules/@epubjs-react-native/core/lib/commonjs/View.js
index 0de0fef..19de579 100644
--- a/node_modules/@epubjs-react-native/core/lib/commonjs/View.js
+++ b/node_modules/@epubjs-react-native/core/lib/commonjs/View.js
@@ -417,6 +417,7 @@ function View({
       uri: templateUri
     },
     showsVerticalScrollIndicator: false,
+    contentMode: "mobile",
     javaScriptEnabled: true,
     originWhitelist: ['*'],
     scrollEnabled: false,
diff --git a/node_modules/@epubjs-react-native/core/lib/module/View.js b/node_modules/@epubjs-react-native/core/lib/module/View.js
index 63f3c24..435d7af 100644
--- a/node_modules/@epubjs-react-native/core/lib/module/View.js
+++ b/node_modules/@epubjs-react-native/core/lib/module/View.js
@@ -408,6 +408,7 @@ export function View({
       uri: templateUri
     },
     showsVerticalScrollIndicator: false,
+    contentMode: "mobile",
     javaScriptEnabled: true,
     originWhitelist: ['*'],
     scrollEnabled: false,
